tests/integration-tests/tests/scheduler_plugin/test_scheduler_plugin/test_scheduler_plugin_integration/pcluster.config.yaml

```diff
@@ -55,10 +55,15 @@ Scheduling:
       PluginResources:
         ClusterSharedArtifacts:
           - Source: https://${Region}-aws-parallelcluster.s3.${Region}.${URLSuffix}/parallelcluster/3.0.0/cookbooks/aws-parallelcluster-cookbook-3.0.0.tgz
+            Checksum: 170de9ec48aa0d8ab2e2666aedd5dc976651b44425cbcf163faf2265211f3c3d
```

Comment:
let's leave this without checksum so that we also have a test without checksum verification

Reply:
Removed